### ISSUE TITLE
Docker for Mac - consistently slow speeds in network.

### ISSUE THREAD

**Issue by a10kiloham** [docker/for-mac#2814]
Tried with latest Stable version and Latest 18.03 Edge version as of this weekend, on High Sierra (and previous OS version Sierra).

Trying docker image linuxserver/sabnzbd on a usenet download. All settings held constant.

On bare metal OSX: approx 44MB/s speeds
On same network Windows PC: approx 44MB/s speeds
On Synology 1515+ on same network in Docker: approx 45MB/s speeds

Docker for Mac: 6-12MB/s speeds. Consistently. Sometimes going as high as 21MB/s. 

I've tried allocating all the cores to Docker for Mac, increasing speed, etc.

Container loaded using :delegated tags for each volume, which makes no difference.

Are there any other ways to troubleshoot this?

**Comment by a10kiloham** [comment]
Gave up, moved to Linux.

**Comment by g13013** [comment]
this issue should be reopened

**Comment by fourgates** [comment]
I have experienced the same issue.

Diagnosis ID: 54A0E2A5-DFEA-4380-8AAC-AABF6C10E66B/20200124232052

I use Docker on my work PC with no issues. I recently started developing on an iMac at home and it takes significantly longer to serve up the same main.js file. No issues when working on the PC. Same code, same images/containers, and I think my iMac actually has a much more powerful processor and more RAM.
